SoftServeInc/jira_data_process
# Cover an app with unit tests (part #1)

### Description

Currently an application doesn't have unit tests. It's very important to cover all app with unit tests on 100%.

**Need to:**
- add jest configuration - ✅;
- add **pre-push** hook to verify, that tests coverage won't decrease after adding new changes - ✅;
- add **pre-commit** hook to verify, if there are no problems in code (eslint and prettier) - ✅;
- cover whole app with unit tests (min 40% of coverage ⚠️).

**Current issue link** - https://github.com/SoftServeInc/jira_data_process/issues/2;

### How to test

1. Run **npm run test** command;
2. Verify, that all app was covered by unit tests.

### Screenshots
<img width="579" alt="image" src="https://user-images.githubusercontent.com/26352406/184789558-727507a8-28e5-429e-8178-c696c5957c9e.png">

### PR Author Checklist
The following are required to be performed and subsequently checked off by the PR Author for all PRs, no exception. Note that, PR Reviewers rely on the "X of Y" display for each pull request in the [Pull request tab](https://github.com/SoftServeInc/jira_data_process/pulls) of this repository, and thus may ignore your PR until your PR shows "3 of 3" in the Pull requests tab for this repo. 


-   [x] Sections of this PR template not filled in or are not applicable have been removed
-   [ ] Existing tests are passing (npm test)
-   [ ] package-lock.json is not part of the changed files if there are no changes to package.json
